baseapp/abci_utils.go

```diff
@@ -133,13 +126,9 @@ func ValidateVoteExtensions(
 		sumVP += vote.Validator.Power
 	}
 
-	if totalVP > 0 {
-		percentSubmitted := math.LegacyNewDecFromInt(math.NewInt(sumVP)).Quo(math.LegacyNewDecFromInt(math.NewInt(totalVP)))
-		if percentSubmitted.LT(VoteExtensionThreshold) {
-			return fmt.Errorf("insufficient cumulative voting power received to verify vote extensions; got: %s, expected: >=%s", percentSubmitted, VoteExtensionThreshold)
-		}
+	if sumVP > (totalVP*2/3)+1 {
```

Comment:
https://github.com/cometbft/cometbft/blob/ec3d9f6ba24f07bd4a9ebf040a61ae21dd8cf72d/types/vote_set.go#L307

Matching comet calculation for reference.

Reply:
should be less than